ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/API.hs

```diff
@@ -306,6 +306,18 @@ data ChainDB m blk = ChainDB {
         -> BlockComponent blk b
         -> m (Follower m blk b)
 
+      -- It might seem natural to have this function also return whether the
```

Comment:
Were you intending for this comment to be part of the Haddock or no? Right now it's just a plain comment, not Haddock.

Whether or not it's Haddock, I'd encourage using single quotes instead of the at-sign. You also used the Markdown backtick, and I think we'd prefer single quotes for that one too.

Reply:
Would you also mention that, at least currently, only testing requires the more information predicate?